config/config-network.yaml

```diff
@@ -173,12 +173,34 @@ data:
     # Knative doesn't know about that otherwise.
     default-external-scheme: "http"
 
+    # internal-encryption is deprecated and replaced by internal-dataplane-trust and internal-controlplane-trust
     # internal-encryption indicates whether internal traffic is encrypted or not.
-    # If this is "true", the following traffic are encrypted:
-    #  - ingress to activator
-    #  - ingress to queue-proxy
-    #  - activator to queue-proxy
     #
     # NOTE: This flag is in an alpha state and is mostly here to enable internal testing
     #       for now. Use with caution.
     internal-encryption: "false"
+
+    # internal-dataplane-trust indicates the level of trust established in the knative data-plane.
```

Comment:
@davidhadas @evankanderson 
Do we need `internal-` prefix for `internal-controlplane-trust` and `internal-dataplane-trust` ? 
I feel `controlplane-trust` and `dataplane-trust` are enough and they implicate "internal".